SonarQube.TeamBuild.Integration/CoverageReportDownloader.cs

```diff
@@ -59,6 +60,10 @@ private void InternalDownloadReport(string tfsUri, string reportUrl, string repo
                         response.Content.CopyToAsync(fileStream).Wait();
                     }
                 }
+                else
+                {
+                    logger.LogError(Resources.PROC_ERROR_FailedToDownloadReportReason, response.StatusCode, response.ReasonPhrase);
```

Comment:
You could test it manually by debugging through the post-processor and changing the report url to one that doesn't exist.